validator.py

```diff
@@ -7,9 +7,10 @@
 
 from collections import deque
 
-class PullBot(PullValidator, GitMerger):
-    config = yaml.load(open("config.yml"))
+from settings import config
 
+class PullBot(PullValidator, GitMerger):
+    config = config
```

Comment:
Other files, such as `utils/merger.py`, access `self.config` and I didn't want to restructure that in case you guys liked it that way. (I'm not a python programmer so I don't know what the best practices are).